### ISSUE TITLE
 Metamask Locked button not prominent on Navbar

### ISSUE THREAD

**Issue by vikaskyadav** [gitcoinco/web#1629]
The metamask button on the navbar in gitcoin is not prominent. The tooltip on hovering on it is also small in size.
![gitcoin1](https://user-images.githubusercontent.com/12607959/42345924-8b5886b2-80be-11e8-99c4-9f34e0f36190.png)

**Comment by thelostone-mc** [comment]
@vikaskyadav we could boldify the text!
We could increase the size but we've got the same info highlighted in red banner below it !

**Comment by vikaskyadav** [comment]
Yeah, I think increasing the font-size and font-weight a little will do.


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality3&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality3&>
